source/ocsp-support/ocsp-support.rst

```diff
@@ -297,11 +300,9 @@ Suggested OCSP Behavior
 For drivers with finer-grain control over their OCSP behavior, the
 suggested OCSP behavior was chosen as a balance between security and
 availability, erring on availability while minimizing network round
-trips. Therefore, in the `Suggested Stapled OCSP
-Behavior <#suggested-ocsp-behavior>`__ and `Suggested Non-Stapled OCSP
-Behavior <#_win0ckq3thfv>`__ sections, drivers are advised not to reach
-out to OCSP endpoints and CRL distribution points to minimize network
-round trips.
+trips. Therefore, in the `Suggested OCSP Behavior <#suggested-ocsp-behavior>`__
```

Comment:
Note: I think when rebasing you'll probably want to take the version in master as I've fixed the broken links using `:refs:` since we have two sections with the same name (albeit at different levels)

Reply:
Will do!